src/applications/simple-forms/21-10210/components/SaveInProgressIntro.jsx

```diff
@@ -0,0 +1,466 @@
+import React from 'react';
```

Comment:
I'm guessing this part is required to make the magic happen where we have pages before it starts counting step x of x?
what is this page responsible for? Just curious.

Reply:
It's required just for things that return JSX.